### PR TITLE
Update to include managed auth and internal marketplace capabilities

### PR DESCRIPTION
**Pull request by Nashaath**
## Purpose
Update to include managed auth and internal marketplace/connection management capabilities

### CODE REVIEW

File: en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-You will build a simple reading list web application with a sign-in page and functionality to interact with a secure backend REST API. You will use Asgardeo, WSO2's Identity as a Service (IDaaS) solution, to secure user authentication to the web application. The application will allow users to sign in and view the reading list. On signing in, a user can add books to a reading list and delete books from the reading list. The application will also allow users to sign out of the application.
+In this quick start guide, you will explore how to expose a service endpoint via Choreo and securely consume the service from a web application. You will use a simple reading list web application with a sign-in page and functionality to interact with a secure backend service. You will also use Choreo's managed authentication to easily set up authentication for your web application without having to dive into the details of security protocols. The application will allow users to sign in and view their reading lists, add books to a reading list, delete books from the reading list, and sign out of the application effortlessly.
```
Comment (shaniR) [inline en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md:5]: effortlessly -> shall we rename this to easily

File: en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md [hunk 2]
```diff
@@ -124,13 +139,11 @@ To test the **Readinglist** REST endpoint via the integrated OpenAPI Console in
...
-You can proceed to the next section to fix the JWT error and publish the service.
-
-## Step 4: Publish the service
+You can proceed to the next section to fix the JWT error and test the service once again.
 
-To fix the JWT error and publish the service, follow the steps given below:
+## Step 5: Fix the JWT error and retest the service
 
-### Step 4.1: Fix the JWT error
+To fix the JWT error and test  the service once again, follow the steps given below:
```
Comment (shaniR) [inline en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md:146]: extra space

File: en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md [hunk 3]
```diff
@@ -174,138 +183,127 @@ To retest the API, follow the steps given below:
...
 
-{% include "create-web-application-qsg.md" %}
+    | **Field**        | **Value**                  |
+    |------------------|----------------------------|
+    | **Name**         | `Reading List Connection`  |
+    | **Description**  | `Connection between the web application and the reading list service`|
+    
+5. Click **Next**. This displays the service URL of the connection for
+each environment the service is deployed in. In this guide, you will see the service URL for the Development environment.
+6. Click **Finish**. This opens the detailed view of the connection you created. You can copy the Service URL displayed here to use when you configure the web application before deploying it.
```
Comment (shaniR) [inline en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md:232]: Shall we put copying the serviceURL as a separate step? It. can be missed otehrwise.

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md (hunk 1 above)
Comment (Nashaath) [reply en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md:5]: Done

File: en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md (hunk 3 above)
Comment (Nashaath) [reply en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md:232]: +1
Done

File: en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md (hunk 2 above)
Comment (Nashaath) [reply en/docs/quick-start-guides/build-your-first-cloud-native-application-with-choreo.md]: Removed extra space.

**Review by shaniR — APPROVED** [review]

**Review by shaniR — APPROVED** [review]

**Review by shaniR — APPROVED** [review]

**Review by DinithiDiaz — APPROVED** [review]

**Pull request merged**